frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.scss

```diff
@@ -0,0 +1,174 @@
+@import "./../../../assets/styles/variables.scss";
+@import "./../../../assets/styles/common.scss";
+
+.shukra-right-container {
+  & .shukra-visual-title {
+    text-transform: uppercase;
+    color: $primary-color;
+    height: 40px;
+    padding: $single-spacing;
+    font-size: $text-size-lg;
+    font-weight: bold;
+    text-align: center;
+  }
+
+  & .shukra-visual-wrapper {
+    height: calc(100vh - 140px);
+    position: relative;
+
+    & .vis-network {
+      outline: none;
+
+      & div.vis-navigation div.vis-button.vis-up,
+      div.vis-navigation div.vis-button.vis-down,
+      div.vis-navigation div.vis-button.vis-left,
+      div.vis-navigation div.vis-button.vis-right,
+      div.vis-navigation div.vis-button.vis-zoomIn,
+      div.vis-navigation div.vis-button.vis-zoomOut,
+      div.vis-navigation div.vis-button.vis-zoomExtends {
+        background-image: none !important;
+        padding: 0px $single-spacing;
+      }
+
+      & div.vis-navigation div.vis-button:hover {
+        box-shadow: none !important;
+      }
+
+      & .vis-button:after {
+        font-size: $text-size-md;
+        color: $secondary-color;
+      }
+
+      & .vis-button:hover:after {
+        font-size: $text-size-md;
+        color: $primary-color;
+      }
+
+      & .vis-button.vis-up:after {
+        content: "▲";
+      }
+
+      & .vis-button.vis-down:after {
+        content: "▼";
+      }
+
+      & .vis-button.vis-left:after {
+        content: "◀";
+      }
+
+      & .vis-button.vis-right:after {
+        content: "▶";
+      }
+
+      & .vis-button.vis-zoomIn:after {
+        content: "+";
+        font-weight: bold;
+        font-size: $text-size-xl;
+      }
+
+      & .vis-button.vis-zoomOut:after {
+        content: "−";
+        font-weight: bold;
+        font-size: $text-size-xl;
+      }
+
+      & .vis-button.vis-zoomExtends:after {
+        content: "⤧";
+        font-weight: bold;
+        font-size: $text-size-xl;
+      }
+
+      & .vis-tooltip {
+        position: absolute;
+        background-color: $primary-color;
+        color: $secondary-color;
+        padding: $single-spacing;
+        font-size: $text-size-xs;
+        border-radius: 2px;
+        margin-left: $single-spacing;
+        text-align: left;
+        box-shadow: 1px 1px 4px 1px $secondary-color;
+        border: 1px solid $primary-color;
+
+        & .status {
```

Comment:
This class used on the left vis-network pane has a nice status dot which could be used for the graph labels as well

Reply:
That was the plan. :'(